cmd/node/main.go

```diff
@@ -157,118 +165,127 @@ VERSION:
 	// Usage: go tool pprof http(s)://ip.of.the.server/debug/pprof/xxxxx
 	profileMode = cli.BoolFlag{
-		Name:  "profile-mode",
-		Usage: "Boolean profiling mode option. If set to true, the /debug/pprof routes will be available on the node for profiling the application.",
+		Name: "profile-mode",
+		Usage: "Boolean profiling mode option. If set to true, the /debug/pprof routes will be available on the node" +
+			" for profiling the application.",
 	}
 	// skIndex defines a flag that specifies the 0-th based index of the private key to be used from initialNodesSk.pem file
 	skIndex = cli.IntFlag{
-		Name:  "sk-index",
-		Usage: "Private key index specifies the 0-th based index of the private key to be used from initialNodesSk.pem file.",
+		Name: "sk-index",
+		Usage: "Private key `index` specifies the 0-th based index of the private key to be used from the PEM file which" +
+			" contains multiple private keys.",
 		Value: 0,
 	}
 	// gopsEn used to enable diagnosis of running go processes
 	gopsEn = cli.BoolFlag{
 		Name:  "gops-enable",
-		Usage: "Enables gops over the process. Stack can be viewed by calling 'gops stack <pid>'",
+		Usage: "If set, will enable gops over the process. Stack can be viewed by calling 'gops stack <pid>'.",
 	}
 	// storageCleanup defines a flag for choosing the option of starting the node from scratch. If it is not set (false)
 	// it starts from the last state stored on disk
 	storageCleanup = cli.BoolFlag{
 		Name:  "storage-cleanup",
-		Usage: "If set the node will start from scratch, otherwise it starts from the last state stored on disk",
+		Usage: "If set the node will start from scratch, otherwise it starts from the last state stored on disk.",
 	}
 
 	// restApiInterface defines a flag for the interface on which the rest API will try to bind with
 	restApiInterface = cli.StringFlag{
 		Name: "rest-api-interface",
-		Usage: "The interface address and port to which the REST API will attempt to bind. " +
+		Usage: "The interface `address and port` to which the REST API will attempt to bind. " +
 			"To bind to all available interfaces, set this flag to :8080",
 		Value: facade.DefaultRestInterface,
 	}
 
 	// restApiDebug defines a flag for starting the rest API engine in debug mode
 	restApiDebug = cli.BoolFlag{
 		Name:  "rest-api-debug",
-		Usage: "Start the rest API engine in debug mode",
+		Usage: "If set, the node will start with the rest API engine in debug mode.",
 	}
 
 	// nodeDisplayName defines the friendly name used by a node in the public monitoring tools. If set, will override
 	// the NodeDisplayName from config.toml
 	nodeDisplayName = cli.StringFlag{
 		Name:  "display-name",
-		Usage: "This will represent the friendly name in the public monitoring tools. Will override the config.toml one",
+		Usage: "This will represent the friendly name in the public monitoring tools. Will override the config.toml one.",
```

Comment:
The user-friendly name for this Node, appearing in the public monitoring tools. Will override the name set in the preferences TOML file.